### ISSUE TITLE
FormBuilderStepper won't support double values

### ISSUE THREAD

**Issue by VOIDCRUSHER** [danvick/flutter_form_builder#182]
It's a common use case to step with constant non-integer absolute values, as in increments of .5 or .25 to avoid loaded dropdown menus. However, using non-integer values in the FormBuilderStepper will throw the following error:

```
The following assertion was thrown building FormField<num>-[LabeledGlobalKey<FormFieldState<dynamic>>#e4387](dirty, dependencies: [_FormScope], state: FormFieldState<num>#9d900):
type 'double' is not a subtype of type 'int'
```
I believe this can be traced all the way up to root SyStepper class used in FormBuilderStepper which declares it's numeric values as int types.

**_In addition to lobbying for this to be fixed, I think this issue should also open a discussion to hopefully remove all  two dependencies on sy_flutter_widgets as the maintainer has signaled there will be no further updates and has no licensing information._** 

I look forward to any further discussion. Thanks!

**Comment by danvick** [comment]
Hi @VOIDCRUSHER,
Thanks for bringing this to my attention. I'll try to make that work ASAP.
I appreciat ethe feedback

**Comment by danvick** [comment]
Now that we don't have our own stepper. I'll consider porting the library to our own library and keep maintaining it.

**Comment by danvick** [comment]
Fixed in v3.7.0. Feel free to reopen in case of any more feedback.

**Comment by VOIDCRUSHER** [comment]
You beat me to a PR I was about to submit. Left some feedback. Thanks for addressing this!
